web/src/jobs/process_article.rs

```diff
@@ -43,11 +43,8 @@ impl Job<AppState> for ProcessArticle {
         .await?;
 
         let markdown = store_in_markdown_table(db, page_snapshot).await?;
-        let markdown_id = markdown.markdown_id;
 
-        generate_and_store_summary(db, markdown_id, &cleaned_html).await?;
-
-        generate_categories(&markdown.summary).await.unwrap();
```

Comment:
We weren't saving these results anywhere so no need to make this call

Reply:
I was calling `generate_categories` so it generates categories as soon the the user inserts a URL onto the Web app. 

And then we were storing the category in the `Category Table`

Reply:
This call I removed I think is just 'duplicative'!

We do the Category stuff in `store_in_markdown_table` and we actually save it to the DB there